io.kavan.GUADEC.json

```diff
@@ -0,0 +1,76 @@
+{
+    "app-id": "io.kavan.GUADEC",
+    "runtime": "org.gnome.Sdk",
+    "runtime-version": "3.38",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.rust-stable"
+    ],
+    "command": "guadec-events",
+     "build-options" : {
+        "append-path" : "/usr/lib/sdk/rust-stable/bin",
+        "env" : {
+            "CARGO_HOME" : "/run/build/guadec-events/cargo",
+            "RUST_BACKTRACE" : "1"
+        }
+    },
+    "cleanup": [
+        "/app/bin/cargo-pkg"
+    ],
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland"
+    ],
+    "modules" : [
+        {
+            "name" : "libhandy",
+            "buildsystem" : "meson",
+            "config-opts" : [
+                "-Dprofiling=false",
+                "-Dintrospection=enabled",
+                "-Dgtk_doc=false",
+                "-Dtests=false",
+                "-Dexamples=false",
+                "-Dvapi=false",
+                "-Dglade_catalog=disabled"
+            ],
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://gitlab.gnome.org/GNOME/libhandy.git",
+                    "tag" : "1.0.0"
+                }
+            ]
+        },
+        {
+            "name" : "cargo-pkg",
+            "buildsystem" : "simple",
+            "build-commands": [
+	            "cargo install cargo-pkg --root /app"
+            ],
+            "cleanup": [
+                "/app/.crates.toml",
+                "/app/.crates2.json",
+                "/app/bin/cargo-pkg"
+            ]
+        },
+        {
+            "name" : "guadec-events",
+            "buildsystem" : "simple",
+            "build-commands": [
+		        "cargo pkg install /app"
```

Comment:
That's how other apps do it with cargo: https://github.com/flathub/com.github.jms55.Sandbox/blob/master/com.github.jms55.Sandbox.json#L26